src/components/views/elements/StyledCheckbox.tsx

```diff
@@ -17,8 +17,10 @@ limitations under the License.
 import React from "react";
 import { randomString } from "matrix-js-sdk/src/randomstring";
 import { replaceableComponent } from "../../../utils/replaceableComponent";
+import classnames from 'classnames';
 
 interface IProps extends React.InputHTMLAttributes<HTMLInputElement> {
+    kind?: string;
```

Comment:
An enum, please, or at least a union type

Reply:
enum would be better, I think.

Reply:
Sounds good, will do. Will we want to do the same for `AccessibleButton` at some point?

Reply:
Yea, we should.